dealership2/src/main/java/main/java/ro/scit/dealership2/Dealership.java

```diff
@@ -0,0 +1,88 @@
+package main.java.ro.scit.dealership2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Dealership {
+    public ArrayList<Car> getListOfCars() {
+        return listOfCars;
+    }
+
+    ArrayList<Car> listOfCars = new ArrayList<>();
```

Comment:
To keep the clean code and readable, member definitions should be before methods.